### ISSUE TITLE
P2: Fix Tooltips not accessible

### ISSUE THREAD

**Issue by yaelleC** [grafana/grafana#56561]
Some of our tooltips are not accessible (you can tab over them to display the help text)

- [x] Cloud monitoring query editor (builder)
- [x] Template variables editor (Fixed by new layout in Grafana 9.3.0)
- [ ] ADX client secret (config editor)
- [x] ~~ADX query builder~~ (Cannot repro)

**Screenshots**
<img width="500" alt="Screenshot 2022-10-14 at 21 30 47" src="https://user-images.githubusercontent.com/42030685/195928865-8b19e5a3-3bd0-41ed-a393-cc3d433d610a.png">

<img width="500" alt="Screenshot 2022-10-14 at 21 38 24" src="https://user-images.githubusercontent.com/42030685/195929003-9d1e291b-35ef-4e2c-b582-0c9678739dce.png">

**Comment by jjromary** [comment]
Hi @yaelleC . You can show me where are these tooltips whith printscreen?

**Comment by yaelleC** [comment]
Sure thing @jjromary , I added them in the description 🙂

**Comment by jjromary** [comment]
@yaelleC  Thank you so much!

**Comment by jjromary** [comment]
@yaelleC I can't reproduce:
- [ ] Template variables editor.

What your version of Grafana?  I'm using 9.3.0 and the option are different than your print screen
![image](https://user-images.githubusercontent.com/37199044/196266155-e659c2cf-195e-44a9-8465-69002d4adc95.png)

**Comment by yaelleC** [comment]
@jjromary , thanks for checking that, indeed I was testing against an older version of Grafana, I'll mark the template variable item as fixed in `9.3.0` 👍
